docs/src/docs/asciidoc/user/serializers-copiers.adoc

```diff
@@ -109,7 +109,7 @@ Implement the following interface, from package `org.ehcache.spi.serialization`:
 
 [source,java,indent=0]
 ----
-include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=21..-1]
+include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=20..-1]
```

Comment:
For some reason, I do not like having these tags leak in API source files ... can other chime in? @AbfrmBlr @albinsuresh @henri-tremblay ?